src/decoder/image.rs

```diff
@@ -447,6 +453,67 @@ impl Image {
 
                 Box::new(Cursor::new(data))
             }
+            CompressionMethod::Fax4 => {
+                let width = u16::try_from(dimensions.0)?;
+                let height = u16::try_from(dimensions.1)?;
+
+                struct Group4Reader<R2> {
```

Comment:
I would like to see this moved into decoder/stream.rs alongside the other decompressors